app.js

```diff
@@ -20,10 +20,20 @@ connect('http://127.0.0.1:8088', 'user', 'pass')
  */
 function clientLoaded (client) {
   client.start('sla');
-  sla(client)
-    .then(console.log)
-    .catch(errHandler)
-    .done();
+  var defer = Q.defer();
+  client.on('StasisStart', function(event, channel) {
+    stasisStart(event, client, channel);
+  });
+}
+
+function stasisStart (event, client, channel) {
+  if(event.args[0] !== 'dialed') {
```

Comment:
That's not actually checking that the argument is numeric. That's just the example given in the standard ARI example. There's no guarantee you'll even see this in a real application. Look for a way to check that the argument is numeric.

What happens if it's not? Right now your application would do nothing (that may be what you intend of course).